test/integ_tests/test_simulator_quantum_task.py

```diff
@@ -46,34 +46,36 @@
 
 # shots-based tests in this file have the capacity to fail rarely due to probabilistic checks.
 # this parameter can be adjusted if we find tests regularly failing.
-SHOTS = 9000
+SHOTS = 2000
 SV1_ARN = "arn:aws:braket:::device/quantum-simulator/amazon/sv1"
 DM1_ARN = "arn:aws:braket:::device/quantum-simulator/amazon/dm1"
-SIMULATOR_ARNS = [SV1_ARN, DM1_ARN]
-ARNS_WITH_SHOTS = [(SV1_ARN, SHOTS), (SV1_ARN, 0), (DM1_ARN, SHOTS), (DM1_ARN, 0)]
+SIMULATOR_ARNS = [SV1_ARN]
+ARNS_WITH_SHOTS = [(SV1_ARN, SHOTS), (SV1_ARN, 0)]
```

Comment:
Any reason to get rid of DM1?

Reply:
Had checked with science a while ago. As the goal is to check the type of tasks vs check the simulators themselves, this was removed. Can add it back in.